docs/sections/user_guide/cli/tools/config/realize-help.out

```diff
@@ -29,7 +29,7 @@ Optional arguments:
   --key-path KEY[.KEY...]
       Dot-separated path of keys to the block to be output
   --values-needed
-      Print report of values needed to render template
+      Print report of values needed to realize config
```

Comment:
I updated `uwtools.cli` so that `config realize` and `template render` pass accurate help messages to `_add_arg_values_needed()` so that the help output fits the mode/action. This output was re-rendered by `make docs` based on that change.